plugin/hosts/hostsfile.go

```diff
@@ -185,8 +185,11 @@ func (h *Hostsfile) parse(r io.Reader, override *hostsMap) *hostsMap {
 }
 
 // ipVersion returns what IP version was used textually
+// gomacro> net.ParseIP("::FFFF:127.0.0.1")
```

Comment:
Please drop the gomacro (no idea what that is) and just tell that `::FFFF` will be detected as v4. Also reference the RFC as. See RFC 4291 Section 2.5.5

Reply:
gomacro: https://github.com/cosmos72/gomacro

Comment updated as requested.